script.service.janitor/addon.xml

```diff
@@ -0,0 +1,70 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<addon id="script.service.janitor" name="Janitor" version="5.0.0" provider-name="Anthirian, drewzh">
+    <requires>
+        <import addon="xbmc.python" version="3.0.0" />
+        <import addon="xbmc.json" version="11.2.0" />
```

Comment:
Please drop from add-on.xml

Reply:
Please drop from add-on xml

Reply:
Not sure exactly what you require me to do with this. Could you please elaborate why this should be removed, or provide me with a changelog for the addon.xml requirements?

Reply:
Ah sorry, review got messed up. I mean that

`<import addon="xbmc.gui" version="5.15.0" />`
and
`<import addon="xbmc.json" version="11.2.0" />`

Aren't needed anymore as dependencies in kodi python addons. The gui dependency is only for skins. The json dependency isn't really needed since the `xbmc.python` hard dependency already enforces a specific kodi version/json-rpc version

Reply:
Right, thanks. That explains a lot. I will omit them then.